vendor/vbom.ml/util/sortorder/natsort.go

```diff
@@ -0,0 +1,76 @@
+package sortorder
+
+// Natural implements sort.Interface to sort strings in natural order. This
+// means that e.g. "abc2" < "abc12".
+//
+// Non-digit sequences and numbers are compared separately. The former are
+// compared bytewise, while the latter are compared numerically (except that
+// the number of leading zeros is used as a tie-breaker, so e.g. "2" < "02")
+//
+// Limitation: only ASCII digits (0-9) are considered.
```

Comment:
Objects that currently accept non-ascii characters (that I'm aware of); 

- networks
- volumes using a volume-driver plugin

So for those we'd have to add some extra logic if we want to add natural sort